cert-processor/src/main/java/org/sunbird/incredible/processor/JsonKey.java

```diff
@@ -26,6 +26,12 @@ public interface JsonKey {
     String CLOUD_UPLOAD_RETRY_COUNT = "CLOUD_UPLOAD_RETRY_COUNT";
     String PRIVATE_CLOUD_STORAGE_SECRET = "PRIVATE_CLOUD_STORAGE_SECRET";
     String PRIVATE_CLOUD_STORAGE_KEY = "PRIVATE_CLOUD_STORAGE_KEY";
+
+    String CLOUD_STORAGE_ENDPOINT = "CLOUD_STORAGE_ENDPOINT";
+    String AZURE="azure";
+    String AWS="aws";
```

Comment:
Do we really need this, can we remove it.

Reply:
i dont think we need it, but i think we added it as there was a similar variable for GCP also. Can you confirm if 
String GCP="gcloud"; 

is also redundant?